### PR TITLE
Add keyboard layouts for homework

### PR DESCRIPTION
**Pull request by m-wynn**
Willy needs this for homework.

I'll push to whatever workstation he's using until it's merged

### CODE REVIEW

File: roles/workstation/tasks/main.yml [hunk 1]
```diff
@@ -98,6 +98,15 @@
...
 
+- name: Install ibus keyboards
+  xbps:
+    pkg:
+      - ibus-hangul
+      - ibus-mozc
+    state: present
+  notify: imodules
+
+
```
Comment (kulinacs) [inline roles/workstation/tasks/main.yml]: Extra newline.

**Review by kulinacs — CHANGES_REQUESTED** [review]

File: roles/workstation/tasks/main.yml (hunk 1 above)
Comment (m-wynn) [reply roles/workstation/tasks/main.yml]: fixup'd

**Review by kulinacs — APPROVED** [review]

**Comment by kulinacs** [comment]
Running test deployment on Oberon now. Will merge if successful.

**Comment by kulinacs** [comment]
`ibus-mozc` is not a package.

**Comment by m-wynn** [comment]
Yes it is?  Are we still syncing properly?

**Comment by dinoocch** [comment]
https://github.com/voidlinux/void-packages/blob/master/srcpkgs/mozc/template

**Comment by m-wynn** [comment]
Closing this, to clean out my GitHub notifications.  Hi guys.

**Pull request closed without merging**